awswrangler/_arrow.py

```diff
@@ -0,0 +1,78 @@
+"""Arrow Utilities Module (PRIVATE)."""
```

Comment:
This module serves two purposes:
1. Collect some arrow specific methods such as converting from a Table to a DataFrame. The objective is to standardise how these operations are done across the codebase
2. Break the circular dependency between the `distributed` module and other `awswrangler` modules. If these methods were to reside in the `_utils` or `s3/_read` module, they would cause circular import dependency issues